docs/csharp/language-reference/compiler-options/command-line-building-with-csc-exe.md

```diff
@@ -30,93 +30,95 @@ translation.priority.ht:
-  
-    ```  
-    csc /target:library File.cs  
-    ```  
-  
--   Compiles File.cs and creates My.exe:  
-  
-    ```  
-    csc /out:My.exe File.cs  
-    ```  
-  
--   Compiles all the C# files in the current directory, with optimizations on and defines the DEBUG symbol. The output is File2.exe:  
-  
-    ```  
-    csc /define:DEBUG /optimize /out:File2.exe *.cs  
-    ```  
-  
--   Compiles all the C# files in the current directory producing a debug version of File2.dll. No logo and no warnings are displayed:  
-  
-    ```  
-    csc /target:library /out:File2.dll /warn:0 /nologo /debug *.cs  
-    ```  
-  
--   Compiles all the C# files in the current directory to Something.xyz (a DLL):  
-  
-    ```  
-    csc /target:library /out:Something.xyz *.cs  
-    ```  
-  
-##  <a name="vcconcommand-linebuildinganchor3"></a> Differences Between C# Compiler and C++ Compiler Output  
- There are no object (.obj) files created as a result of invoking the C# compiler; output files are created directly. As a result of this, the C# compiler does not need a linker.  
-  
-## See Also  
- [C# Compiler Options](../../../csharp/language-reference/compiler-options/index.md)   
- [C# Compiler Options Listed Alphabetically](../../../csharp/language-reference/compiler-options/listed-alphabetically.md)   
- [C# Compiler Options Listed by Category](../../../csharp/language-reference/compiler-options/listed-by-category.md)   
- [Main() and Command-Line Arguments](../../../csharp/programming-guide/main-and-command-args/index.md)   
- [Command-Line Arguments](../../../csharp/programming-guide/main-and-command-args/command-line-arguments.md)   
- [How to: Display Command Line Arguments](../../../csharp/programming-guide/main-and-command-args/how-to-display-command-line-arguments.md)   
- [How to: Access Command-Line Arguments Using foreach](../../../csharp/programming-guide/main-and-command-args/how-to-access-command-line-arguments-using-foreach.md)   
- [Main() Return Values](../../../csharp/programming-guide/main-and-command-args/main-return-values.md)
\ No newline at end of file
+>  When you build a project by using the Visual Studio IDE, you can display the **csc** command and its associated compiler options in the **Output** window. To display this information, follow the instructions in [How to: View, Save, and Configure Build Log Files](/visualstudio/ide/how-to-view-save-and-configure-build-log-files) to change the verbosity level of the log data to **Normal** or **Detailed**. After you rebuild your project, search the **Output** window for **csc** to find the invocation of the C# compiler.
+
+ **In this topic**
+
+- [Rules for Command-Line Syntax](#vcconcommand-linebuildinganchor1)
+
+- [Sample Command Lines](#vcconcommand-linebuildinganchor2)
+
+- [Differences Between C# Compiler and C++ Compiler Output](#vcconcommand-linebuildinganchor3)
+
+##  <a name="vcconcommand-linebuildinganchor1"></a> Rules for Command-Line Syntax for the C# Compiler
```

Comment:
![capture](https://cloud.githubusercontent.com/assets/1622880/25264698/41b44154-262e-11e7-97f7-faf71d4d9c89.PNG)

My hunch was correct. We should get the anchors out.